examples/RetrieveAuditEvents.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
+'''
+This script retrieves all audit logs across an OCI Tenancy
+for a defined timespan.
+
+This script relies on OCI config as per the format defined at
+https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm
+
+This script will work at a tenancy level only.
+
+The config file should have the following contents
+User OCID
+RSA private key in PEM format
+Fingerprint
+Tenancy OCID
+'''
+
+import oci
+
+__author__ = "saurabh.bangad@oracle.com"
+
+configfile = "~/.oci/config"
+
+config = oci.config.from_file(configfile)  # Path to relevant config file
+tenancy_id = config["tenancy"]
+# This array will be used to store the list of compartments in the tenancy
+compartments = []
+compartments.append(tenancy_id)  # Tenancy is the first compartment
+regions = []  # This array will be used to store the list of available regions
+auditevents = []
+
+
+#  Initiate the client with the locally available config
+#  These are global variables and used across all the functions
+audit = oci.audit.audit_client.AuditClient(config)
+identity = oci.identity.IdentityClient(config)
+
+
+# Timespan defined by variables start_time and end_time
+start_time = '2018-01-15T00:00:00Z'
+end_time = '2018-01-17T00:00:00Z'
+
+
+def get_regions():
+    '''To retrieve the list of all available regions
+    '''
+    listofregions = []
```

Comment:
I'll stop commenting on it after this, but please review your variables and use `snake_case` rather than just running all the words together, e.g. this variable would be `list_of_regions`